projects/fsastorefrontlib/src/cms-components/my-account/quote/quote-comparison/quote-comparison.component.html

```diff
@@ -103,8 +95,26 @@ <h2 class="heading-headline p-3 mt-3 mb-5 text-center">
                 {{
                   ((cart?.deliveryOrderGroups)[0]?.entries)[0]?.product?.name
                 }}
+                <span
+                  *ngIf="
+                    !accordionWrapper.firstChild.tagName &&
```

Comment:
This is a condition I used to differentiate the event case.

Reply:
Shouldn't we use `categoryConfig?.visibleInsuredObjects?.length > 0` in that case? Maybe that would make it more obvious for what you are suggesting. As it is now I would assume it's there to check if it's accordion is the first one.